docs/options.md

```diff
@@ -393,6 +393,14 @@ Note that “in tandem” doesn’t mean “at the same time”. When the two op
 
 [adoption strategy]: https://prettier.io/blog/2017/05/03/1.3.0.html#facebook-adoption-update
 
+## Check Ignore Pragma
+
+Whether or not to detect and respect `@noprettier` or `@noformat` markers at the top of files. These will prevent formatting when this option is enabled.
```

Comment:
We should mention the reason why this is disabled by default.

> My thinking here was mainly around performance

Reply:
Rewrote this options.md section to include the rationale.